httpcore/_backends/asyncio.py

```diff
@@ -131,9 +131,21 @@ async def read(self, n: int, timeout: TimeoutDict) -> bytes:
         exc_map = {asyncio.TimeoutError: ReadTimeout, OSError: ReadError}
         async with self.read_lock:
             with map_exceptions(exc_map):
-                return await asyncio.wait_for(
-                    self.stream_reader.read(n), timeout.get("read")
-                )
+                try:
+                    return await asyncio.wait_for(
+                        self.stream_reader.read(n), timeout.get("read")
+                    )
+                except AttributeError as ex:  # pragma: nocover
+                    if "resume_reading" in str(ex):
+                        # We have to return empty string there,
+                        #     due to one ugly bug in asyncio
+                        # More information you can find in this issue:
+                        #     https://github.com/encode/httpx/issues/1213
+                        #
+                        # The empty byte-string returned from there caused
+                        #     httpcore.RemoteProtocolError at the top level
```

Comment:
(Nit) Tweaking the leading spaces here…

```suggestion
                        # We have to return empty string there,
                        # due to one ugly bug in asyncio
                        # More information you can find in this issue:
                        # https://github.com/encode/httpx/issues/1213
                        #
                        # The empty byte-string returned from there caused
                        # httpcore.RemoteProtocolError at the top level
```

Reply:
Fixed